### ISSUE TITLE
Using the service mirror causes OOM

### ISSUE THREAD

**Issue by gyz2009** [containous/traefik#6125]
### **Do you want to request a feature or report a bug?**
bug

### **What did you do?**
enable service mirror and HTTP performance test

### **What did you expect to see?**
Traefik not error

### **What did you see instead?**
![image](https://user-images.githubusercontent.com/30786213/71714629-98d7ef00-2e49-11ea-8fd5-63f5a3cc85c7.png)

![image](https://user-images.githubusercontent.com/30786213/71714486-0e8f8b00-2e49-11ea-95d0-ef60c867e9f8.png)

Then I made a million HTTP requests and stopped, and the memory wasn't reclaimed
![image](https://user-images.githubusercontent.com/30786213/71714728-f5d3a500-2e49-11ea-8e07-74885be68c04.png)

### **Output of traefik version:**
Traefik viersion:  v2.1

### **What is your environment & configuration (arguments, toml, provider, platform, ...)?**
OS: CentOS 7.6  
Kernel: 3.10.0-957.21.3.el7
Static Config
```
entryPoints:
  web:
    address: ":80"

  websecure:
    address: ":443"

providers:
# Enable the file provider to define routers/middlewares/services in file
  file:
    directory: /etc/traefik/conf
```

Provider Config:
```
http:
  routers:
    to-whoami-https:
      rule: "Host(`aaa.com`) && PathPrefix(`/`)"
      entryPoints:
      - "websecure"
      service: mirrored-api
      tls: {}
    to-whoami-http:
      rule: "Host(`aaa.com`) && PathPrefix(`/`)"
      entryPoints:
      - "web"
      service: mirrored-api

  services:
    mirrored-api:
      mirroring:
        service: appv1
        mirrors:
        - name: appv2
          percent: 100

    appv1:
      loadBalancer:
        servers:
        - url: http://xxxxa/

    appv2:
      loadBalancer:
        servers:
        - url: http://xxxxxb/
tls:
  certificates:
    - certFile: /etc/traefik/ssl/test.pem
      keyFile: /etc/traefik/ssl/test.key
      stores:
        - default
```

**Comment by mpl** [comment]
@gyz2009 hi. We cannot help if we cannot reproduce the issue. And we cannot reproduce the issue faithfully if we do not have your static config. Could you please provide your full configuration?

**Comment by gyz2009** [comment]
@mpl  The static config has been updated

**Comment by mpl** [comment]
@gyz2009 I can reproduce, thank you.

**Comment by traefiker** [comment]
Closed by #6140.
